icon.dust

```diff
@@ -1,3 +1,11 @@
-<div {?id}id="{.}"{/id} class="c-icon {class}" {?role}role="{.}"{/role}>
-    Component stuff goes here!
-</div>
+<svg class="c-icon {class}"
+    {?id}id="{id}"{/id}
+    {?title}title="{title}"{/title}
```

Comment:
Isn't the `title` an inaccessible attribute? As far as I'm aware, screen readers use it inconsistently and is better used as mouse-over text. Perhaps `label`, as in `aria-label`, would be a better option. Although there is the issue that `aria-label` doesn't work like we thought it does... I don't think we tested it on SVG elements, specifically.